src/api/goods.js

```diff
@@ -6,6 +6,12 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => getAll()
+  .then(goods => (
+    goods.sort((a, b) => a.name.localeCompare(b.name)).splice(0, 5)
+  ));
 
-export const getRedGoods = () => {};
+export const getRedGoods = () => getAll()
+  .then(goods => (
+    goods.filter(good => good.color === 'red').splice(0, 5)
```

Comment:
You don't need splice here. This function should get all the red goods, not only 5 of them